src/Model.elm

```diff
@@ -61,6 +63,13 @@ init flags location =
         , currentView = LocalTimelineView
         , currentUser = Nothing
         , notificationFilter = NotificationAll
+        , autoState = Autocomplete.empty
+        , autoAtPosition = Nothing
+        , autoQuery = ""
+        , autoCursorPosition = 0
+        , autoMaxResults = 4
+        , autoAccounts = []
```

Comment:
Could it be possible to put these into a new field of the `draft` one? or maybe at least into a single `autocomplete` one?